main.go

```diff
@@ -101,13 +101,16 @@ func run() int {
 	cmd.Stdin = os.Stdin
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
-	err = cmd.Run()
 
+	err = cmd.Run()
 	code := cmd.ProcessState.ExitCode()
-	if code == 0 {
-		println(string(bk))
+	if code == 0 && err == nil {
+		if verbose {
+			fmt.Fprintf(os.Stderr, "[done] %v\n", string(b))
+		}
 		if err = store.Delete(bk, &opt.WriteOptions{Sync: true}); err != nil {
 			fmt.Fprintf(os.Stderr, "%v: %v\n", os.Args[0], err)
+			return -2
```

Comment:
exit code should be unsigned int

Reply:
pls check:
```
// ExitCode returns the exit code of the exited process, or -1
// if the process hasn't exited or was terminated by a signal.
```
tho i am not completely sure how to make it happen, :-)